### PR TITLE
Adds some more utility functions for Pods

### PR DESCRIPTION
**Pull request by lfrancke**

### CODE REVIEW

File: src/podutils.rs [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+    Unknown,
+}
+
+impl fmt::Display for PodPhase {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        fmt::Debug::fmt(self, f)
+    }
+}
+
+pub enum PodConditionType {
```
Comment (soenkeliebau) [inline src/podutils.rs:20]: Is there a specific reason that Debug and Display are not derived/implemented for this?

**Review by soenkeliebau — APPROVED** [review]
LGTM, left just one minor question

File: src/podutils.rs (hunk 1 above)
Comment (lfrancke) [reply src/podutils.rs:20]: It's not actually used anywhere yet....I need to figure out how to use it and then it makes sense to add that, yes.

**Review by soenkeliebau — APPROVED** [review]
LGTM

**Pull request merged**